docs/high/dataset.rst

```diff
@@ -383,6 +383,14 @@ Reference
             >>> arr = np.zeros((100,), dtype='int32')
             >>> dset.read_direct(arr, np.s_[0:10], np.s_[50:60])
 
+  .. method:: write_direct(self, source, source_sel=None, dest_sel=None):
```

Comment:
This indentation does not match the indentation above and blow, I suspect this makes it not render correctly.